### PR TITLE
Prose updates for 2019q2 report

### PR DESCRIPTION
**Pull request by zeising**
This looks great, I tried to rework the section on debugfs, but I would like @johalun to take a look at this make sure I'm being accurate.

### CODE REVIEW

**Comment by zeising** [comment]
@nomadlogic , can you reformat so that lines wrap at 80 characters and setences start on new lines?
We'll get @johalun a chance to comment as well.

**Comment by zeising** [comment]
@nomadlogic I made the line wrap changes and stuff like that. The quarterly report people wants this ASAP, so I went ahead and made the changes
I'll try to make @johalun chime in as well, but I have tp push this tonight.

File: 2019q2/graphics.md [hunk 1]
```diff
@@ -56,7 +57,8 @@ Several userland Xorg drivers, applications, and libraries have been updated, an
 other improvements to the various userland components that make up the Graphics
 Stack have been made.
 
-Several meetings have been held over the course of the period.
+We have also continued our regularlly scheduled bi-weekly meetings, although work
```
Comment (johalun) [inline 2019q2/graphics.md]: Typo `regularlly`

Comment (johalun) [reply 2019q2/graphics.md]: otherwise lgtm

**Pull request merged**